packages/design-system-react-native/src/components/Button/variants/ButtonPrimary/ButtonPrimary.tsx

```diff
@@ -1,6 +1,6 @@
 import {
   useTailwind,
-  withThemeProvider,
+  ThemeProvider,
```

Comment:
Certainly! Here’s a clear and concise comment you can add to the PR for context:

---
This change removes the withThemeProvider HOC in favor of directly using the ThemeProvider component to lock ButtonPrimary to the light theme. Using ThemeProvider inline is more idiomatic in modern React, makes theme logic clearer at the usage site, and avoids the need for extra wrapper components. This also improves maintainability and aligns with current best practices for context management.